Formula/l/llgo.rb

```diff
@@ -0,0 +1,45 @@
+class Llgo < Formula
+  desc "Go compiler based on LLVM integrate with the C ecosystem and Python"
+  homepage "https://github.com/goplus/llgo"
+  url "https://github.com/goplus/llgo/archive/refs/tags/v0.8.8.tar.gz"
+  sha256 "be6979d47bcc8f89f2156f6b2d6f603854104fea7452519bac144400fbecabf0"
+  license "Apache-2.0"
+
+  depends_on "go"
+  depends_on "llvm@17"
```

Comment:
any issue tracking llvm18 support?

Reply:
Current version only works with llvm@17, will fix in next release.

Reply:
when would be the next release? maybe better wait for that one?

Reply:
OK, will update after next major release. But maybe need lock to `llvm@18`.

Reply:
I actually dont think we are able to pin the llvm version to 18 (maybe good left a note to update to it once llvm 19 is out)

now we are talking about this, I feel like we can just ship with llvm 17